fixtures/storybook-config/src/TestComponent.tsx

```diff
@@ -1,15 +1,30 @@
-import { Card, Text } from 'braid-design-system';
+import { Box, Card, Text } from 'braid-design-system';
 import React from 'react';
+// @ts-expect-error
+import { Text as SSGText, HeartIcon } from 'seek-style-guide/react';
```

Comment:
Thoughts on `import * as ssg`? Then you could write `<ssg.HeartIcon />`, `<ssg.Text />` etc.